classes/kernel.oeclass

```diff
@@ -15,10 +15,14 @@
 ##          DEPENDS_KERNEL_INITRAMFS to acquire the initramfs.
 ## @useflag ramdisk_image_compression Compression of the ramdisk image. Set to
 ##          none by default.
-## @useflag kernel_dtb Name of the compiled Device tree source (dts).
-## @useflag kernel_dtc Whether to use the linux device tree compiler (dtc).
+## @useflag kernel_dtb Name of the (already compiled) Device tree binary (dtb)
```

Comment:
ok so the make target that is automatically there when the corresponding .dts is present?

Think those two things should be seperated then, I do see a use for specifying a .dtb directly as well anyway